lang/java/reef-common/src/main/java/org/apache/reef/client/DriverLauncher.java

```diff
@@ -59,15 +60,21 @@
       .build();
 
   private final REEF reef;
+  private final UserCredentials user;
 
   private LauncherStatus status = LauncherStatus.INIT;
 
   private String jobId;
   private RunningJob theJob;
 
   @Inject
-  private DriverLauncher(final REEF reef) {
+  private DriverLauncher(final REEF reef, final UserCredentials user) {
```

Comment:
So this is the user that is used to set to the YarnProxyUser in the Environment, right? I just curious where the initial value of this user comes  from. How it is related to the User at client side? Do they have the same tokens?